src/Sylius/Component/Shipping/Checker/ShippingMethodEligibilityCheckerInterface.php

```diff
@@ -13,10 +13,19 @@
 
 namespace Sylius\Component\Shipping\Checker;
 
+use const E_USER_DEPRECATED;
+use Sylius\Component\Shipping\Checker\Eligibility\ShippingMethodEligibilityCheckerInterface as NewShippingMethodEligibilityCheckerInterface;
 use Sylius\Component\Shipping\Model\ShippingMethodInterface;
 use Sylius\Component\Shipping\Model\ShippingSubjectInterface;
 
-interface ShippingMethodEligibilityCheckerInterface
+@trigger_error(sprintf('The "%s" interface is deprecated since Sylius 1.7, use "%s" instead.', ShippingMethodEligibilityCheckerInterface::class, NewShippingMethodEligibilityCheckerInterface::class), E_USER_DEPRECATED);
+
+/**
+ * @deprecated since Sylius 1.7. Use Sylius\Component\Shipping\Checker\Eligibility\ShippingMethodEligibilityCheckerInterface instead
```

Comment:
```suggestion
 * @deprecated since Sylius 1.8. Use Sylius\Component\Shipping\Checker\Eligibility\ShippingMethodEligibilityCheckerInterface instead
```